modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1578,6 +1588,9 @@ public ObjectEntry updateObjectEntry(
 			ObjectEntryThreadLocal.setSkipObjectValidationRules(false);
 		}
 
+		_updateResourcePermissions(
+			objectDefinition, objectEntry, serviceContext);
+
```

Comment:
I'm wondering about the order in which this method is called.
In the ADD method, you call this method after` _updateAsset(...)` and here it is before.
Is there a required order for these methods to work?

Reply:
@marcelabc It was my mistake while fixing the rebase conflict. I'll change it to put it before `_updateAsset`